src/App.css

```diff
@@ -43,6 +45,14 @@ body {
   border-radius: 5rem;
 }
 
+/* 
+    to hide "reveal" button at the edge of a password <input> which 
+    displays the value of the password field in plain text 
+*/
+::-ms-reveal {
+  display: none !important;
+}
+
```

Comment:
Query - So this CSS rule is for Microsoft edge, right? But your comment can be misunderstood. Also, I don't understand the purpose of this can you elaborate on this?

Reply:
Yeah, so on edge browser it shows a button to hide/show password by default. But as I am giving this option, explicitly, so it was showing two buttons for the same thing.

By this code, I am hiding the default one.

Reply:
Got it thanks.